### ISSUE TITLE
doc folder to deploy to gh_pages

### ISSUE THREAD

**Issue by y-nk** [greiman/SdFat#494]
I think it's a shame that you have a nice doc folder as html but not hosted somewhere (at least I couldn't find the online version). You could leverage github pages hosting, which i think is quite easy. I can help provide guidance or more ; would you like that?

**Comment by greiman** [comment]
> You could leverage github pages hosting, which i think is quite easy.

Yes I have thought about doing it but I don't even have time to test and post bug fixes.  Also I have new support for SDIO on RP2040/RP2350 that does read/write at over 20 MB/sec but I have not had time to finish testing it.
